src/app/game/game.component.ts

```diff
@@ -33,10 +37,14 @@ export interface Game {
     `
   ]
 })
+
+//Je comprends pas bien ce que ça fait // c'est la que on mettera le service pour récupérer les game data ?
+// WHy does GameComponent has @input decorator ?
```

Comment:
The `GameComponent` will be responsible to display and have the game run.
To build a `Game` object, an other component should be used. This component will walk the player through all the available options and emit a `Game` object when the player notifies he is ready.

That's why the `GameComponent` has the `Game` as input.